opentelemetry-instrumentation/src/opentelemetry/instrumentation/bootstrap_gen.py

```diff
@@ -53,8 +53,8 @@
         "instrumentation": "opentelemetry-instrumentation-celery==0.40b0.dev",
     },
     "confluent-kafka": {
-        "library": "confluent-kafka >= 1.8.2, < 2.0.0",
-        "instrumentation": "opentelemetry-instrumentation-confluent-kafka==0.40b0.dev",
+        "library": "confluent-kafka >= 1.8.2, <= 2.1.1",
+        "instrumentation": "opentelemetry-instrumentation-confluent-kafka==0.39b0.dev",
```

Comment:
It looks like a merge issue with the opentelemetry-instrumentation-confluent-kafka version